app/forms/provider_interface/recruit_with_pending_conditions_form.rb

```diff
@@ -0,0 +1,26 @@
+module ProviderInterface
+  class RecruitWithPendingConditionsForm
+    include ActiveModel::Model
+
+    attr_accessor :actor, :application_choice, :confirmation
+
+    validates :confirmation, presence: true
+    validates :confirmation, inclusion: { in: %w[yes no] }
+
+    def save
+      return false unless valid?
+
+      # TODO: Work out whether we really need the radio buttons here and hence
+      # whether we even need the `confirmation` attribute here.
+      if confirmed?
+        ConfirmOfferConditions.new(actor:, application_choice:, updated_conditions: false).save
```

Comment:
Need to check the `updated_conditions` option here (doesn't look like it's implemented by the service).

Reply:
Consider adding a new more specific service (with a different state machine event).

Reply:
agree – could we use a separate service for this, thanks!